### PR TITLE
Fix broken tests

### PR DESCRIPTION
**Pull request by rastamhadi**
Several tests are broken on [Travis](https://travis-ci.org/github/vividmuimui/lock_diff/jobs/581970224).

### CODE REVIEW

File: spec/lock_diff/github/repository_name_detector_spec.rb [hunk 1]
```diff
@@ -5,7 +5,7 @@ module LockDiff::Github
     specify do
       expect(RepositoryNameDetector.new("https://github.com/rr/rr").call).to eq 'rr/rr'
       expect(RepositoryNameDetector.new("https://github.com/rr/rr/foo/bar/baz").call).to eq 'rr/rr'
-      expect(RepositoryNameDetector.new("https://github.com/rr/rr/foo#readme").call).to eq 'rrg/rr'
+      expect(RepositoryNameDetector.new("https://github.com/rr/rr/foo#readme").call).to eq 'rr/rr'
```
Comment (rastamhadi) [inline spec/lock_diff/github/repository_name_detector_spec.rb:8]: I went ahead and assumed that this was a typo in #41.

File: lock_diff.gemspec [hunk 2]
```diff
@@ -26,7 +26,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "octokit", "~> 4.0"
   spec.add_dependency "httpclient"
 
-  spec.add_development_dependency "bundler", "~> 1.15"
+  spec.add_development_dependency "bundler", "> 1.15"
```
Comment (rastamhadi) [inline lock_diff.gemspec:29]: I made this change after running into the following error when running `bundle install` with Bundler v2.1.4.

```console
$ bundle
Fetching gem metadata from https://rubygems.org/........
Fetching gem metadata from https://rubygems.org/.
Resolving dependencies...
Bundler could not find compatible versions for gem "bundler":
  In Gemfile:
    bundler (~> 1.15)

  Current Bundler version:
    bundler (2.1.4)
This Gemfile requires a different version of Bundler.
Perhaps you need to update Bundler by running `gem install bundler`?

Could not find gem 'bundler (~> 1.15)' in any of the relevant sources:
  the local ruby installation
```

**Comment by rastamhadi** [comment]
[Six of the tests](https://travis-ci.org/github/vividmuimui/lock_diff/jobs/731850218) are still failing with

```
Failure/Error: super(token || ENV.fetch('GITHUB_ACCESS_TOKEN'))

KeyError:
  key not found: "GITHUB_ACCESS_TOKEN"
# ./lib/lock_diff/github/access_token.rb:5:in `fetch'
```

It looks like Travis isn't setting the `GITHUB_ACCESS_TOKEN` environment variable. @vividmuimui will probably have to look into this since he is the only one with access to the Travis environment variables.

**Comment by vividmuimui** [comment]
Thanks!

---

> It looks like Travis isn't setting the GITHUB_ACCESS_TOKEN environment variable. @vividmuimui will probably have to look into this since he is the only one with access to the Travis environment variables.

I set up the environment variables in travis. And now the test passes on travis CI with PR #45.

**Pull request merged**